ivy_tests/test_ivy/test_frontends/test_torch/test_dropout_functions.py

```diff
@@ -50,3 +50,48 @@ def test_torch_dropout(
     for u in ret:
         # cardinality test
         assert u.shape == x.shape
+
+
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.dropout2d",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=50,
+        allow_inf=False,
+        min_num_dims=2,
```

Comment:
The `min_num_dims` needs to be `3` since `torch` supports either `NCL` or `NCHW` in the input